spec/models/event_instance_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe EventInstance do
```

Comment:
This spec needs to validate the uniqueness of an instance year scoped to the parent event